otter/assign/assignment.py

```diff
@@ -16,6 +16,13 @@ class Assignment(fica.Config, Loggable):
     Configurations for the assignment.
     """
 
+
+    traceback_length: Optional[str] = fica.Key(
```

Comment:
nit: please add new configurations at the bottom of the `Key`s (i.e. after `runs_on`)